src/js/components/StarRating/StarRating.js

```diff
@@ -0,0 +1,41 @@
+import React, { useContext, useState } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Star } from 'grommet-icons/icons/Star';
+import { StarOutline } from 'grommet-icons/icons/StarOutline';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+const StarRating = ({ ...rest }) => {
+  const theme = useContext(ThemeContext);
+  const [value, setValue] = useState();
+
+  const options = [];
+  for (let i = 1; i < 6; i += 1) {
+    options.push(i);
+  }
+
+  return (
+    <RadioButtonGroup
+      direction="row"
+      options={options}
+      onChange={(event) => {
+        setValue(event.target.value);
+      }}
+      {...rest}
+    >
+      {(option) =>
+        option <= value ? (
+          <Star
+            color={theme.starRating?.color || theme.starRating?.color?.fill}
```

Comment:
This use of `theme.starRating.color.fill` doesn't align with the typescript definition, where `ColorType` is just a string or an object with dark/light keys. Can we simplify this to just `theme.starRating.color`?

Reply:
aw yes I see your point wondering if I should make `theme.starRating.color.fill` its own type so we can have `theme.starRating.color` where someone can pass dark/light keys and also have the `theme.starRating.color.fill` and `theme.starRating.color.outline` if they want different colors depending on which icon is selected or you saying for now just allow one color for the icons regardless of fill out outline?

Reply:
I think the theme should just be asking for a color and we should use it appropriately within the StarRating to apply the color where people would typically expect it, as outline for unselected, as fill for selected.